spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -71,6 +71,135 @@
+        expect(response[:attributes][:work_experience][:work_history_break_explanation]).to eq(
+          "February 2019 to April 2019: I was watching TV.\n\nSeptember 2019 to December 2019: I was playing games.",
+        )
+      end
+    end
+
+    context 'when the work breaks feature flag is on and no breaks have been entered' do
+      it 'returns an empty string' do
+        FeatureFlag.activate('work_breaks')
+        breaks = []
+        application_form = build_stubbed(
+          :completed_application_form,
+          :with_completed_references,
+          work_history_breaks: nil,
+          application_work_history_breaks: breaks,
+        )
+        application_choice = build_stubbed(:application_choice, status: 'awaiting_provider_decision', application_form: application_form)
+
+        response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
+
+        expect(response.to_json).to be_valid_against_openapi_schema('Application')
+        expect(response[:attributes][:work_experience][:work_history_break_explanation]).to eq('')
+      end
+    end
+
+    context 'when the work breaks feature flag is off and no breaks have been entered' do
+      it 'returns an empty string' do
+        FeatureFlag.deactivate('work_breaks')
+        breaks = []
+        application_form = build_stubbed(
+          :completed_application_form,
+          :with_completed_references,
+          work_history_breaks: nil,
+          application_work_history_breaks: breaks,
+        )
+        application_choice = build_stubbed(:application_choice, status: 'awaiting_provider_decision', application_form: application_form)
+
+        response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
+
+        expect(response.to_json).to be_valid_against_openapi_schema('Application')
+        expect(response[:attributes][:work_experience][:work_history_break_explanation]).to eq('')
+      end
+    end
+
+    context 'when the work breaks feature flag is off and work history breaks field has a value' do
+      it 'returns the work_history_breaks attribute of an application' do
+        FeatureFlag.deactivate('work_breaks')
+        breaks = []
+        application_form = build_stubbed(
+          :completed_application_form,
+          :with_completed_references,
+          work_history_breaks: 'I was sleeping.',
+          application_work_history_breaks: breaks,
+        )
+        application_choice = build_stubbed(:application_choice, status: 'awaiting_provider_decision', application_form: application_form)
+
+        response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
+
+        expect(response.to_json).to be_valid_against_openapi_schema('Application')
+        expect(response[:attributes][:work_experience][:work_history_break_explanation]).to eq('I was sleeping.')
+      end
+    end
+
+    context 'when the work breaks feature flag is off and work history breaks field is empty' do
```

Comment:
Great specs. I think these three could be made a bit more distinct, maybe:

- `when the work breaks feature flag is off and no breaks have been entered`
- `when the work breaks feature flag is off and the work history breaks field has a value`
- `when the work breaks feature flag is off and breaks have been entered`

ie when there are breaks, say what sort they are rather than what sort they aren't